README.adoc

```diff
@@ -351,6 +351,20 @@ class User
 end
 user&.zip
 ----
+
+=== Long safe navigation chain [[long-safe-navigation-chain]]
+
+Avoid `&.` chains longer than two.
+
+[source,ruby]
+----
+# bad
+user&.address&.zip
```

Comment:
I'm unsure if this example is appropriate. According to https://github.com/rubocop/rubocop/pull/13228, up to two consecutive `&.` are allowed by default. It would be better to explicitly clarify, with an explanation and example code, how many consecutive `&.` are considered "Long safe navigation chain".